docs/concepts/jcloud/index.md

```diff
@@ -482,6 +482,219 @@ jc job logs myjob1 -f rich-husky-af14064067
 :width: 90%
 ```
 
+## Deployments
+
+### Deploy
+
+In Jina's idiom, a project is a [Deployment](https://docs.jina.ai/concepts/orchestration/Deployment/), which represents an end-to-end task such as indexing, searching or recommending. In this document, we use "project" and "Deployment" interchangeably.
+
+```{caution}
+When `jcloud` deploys a deployment it automatically appends the following global arguments to the `deployment.yml`, if not present:
+```
+
+```yaml
+jcloud:
+  version: jina-version
+  docarray: docarray-version
+```
+
+#### Single YAML file
+
+A self-contained YAML file, consisting of all configuration at the [Deployment](https://docs.jina.ai/concepts/orchestration/deployment/)-level and [Executor](https://docs.jina.ai/concepts/serving/executor/)-level.
+
+> A Deployment's `uses` must follow the format `jinaai+docker://<username>/MyExecutor` (from [Executor Hub](https://cloud.jina.ai)) to avoid any local file dependencies:
```

Comment:
```suggestion
> A Deployment's `uses` parameter must follow the format `jinaai+docker://<username>/MyExecutor` (from [Executor Hub](https://cloud.jina.ai)) to avoid any local file dependencies:
```